src/utils/translations/vi.ts

```diff
@@ -0,0 +1,94 @@
+const vi = {
+  // Home Tab
+  Home: "Trang chủ",
+  "Period in": "Kỳ kinh nguyệt trong vòng",
+  "no info": "không có thông tin",
+  Period: "Kỳ kinh nguyệt",
+  today: "hôm nay",
+  Delay: "Bị chậm trễ",
+  Days: "(0)[ngày];(1)[ngày];(2-4)[ngày];(5-20)[ngày];(21)[ngày];(22-24)[ngày];(25-30)[ngày];(31)[ngày];(32-34)[ngày];(35-40)[ngày];",
```

Comment:
You can use this instead

```ts
  Days: "(0-inf)[ngày]",
```

Reply:
Thank you for your review. I have updated the translations.